ffmpeg/_run.py

```diff
@@ -93,11 +93,16 @@ def _get_output_args(node, stream_name_map):
     if node.name != output.__name__:
         raise ValueError('Unsupported output node: {}'.format(node))
     args = []
-    assert len(node.incoming_edges) == 1
-    edge = node.incoming_edges[0]
-    stream_name = stream_name_map[edge.upstream_node, edge.upstream_label]
-    if stream_name != '[0]':
-        args += ['-map', stream_name]
+
+    if len(node.incoming_edges) == 0:
+        raise ValueError("Output node {} has no mapped streams")
+
+    for edge in node.incoming_edges:
+        # edge = node.incoming_edges[0]
+        stream_name = "[{}{}]".format(stream_name_map[edge.upstream_node, edge.upstream_label], "" if not edge.upstream_selector else ":{}".format(edge.upstream_selector))
```

Comment:
This line is too complicated. See comment about `_format_output_stream_name` above.

Reply:
i.e.
```diff
-       stream_name = "[{}{}]".format(stream_name_map[edge.upstream_node, edge.upstream_label], "" if not edge.upstream_selector else ":{}".format(edge.upstream_selector))
+       stream_name = _format_input_stream_name(stream_name_map, edge)
```

Reply:
Done 21bd1cd8